.github/workflows/release_zip.yaml

```diff
@@ -0,0 +1,53 @@
+name: Release Branch Update
+
+on: [release, workflow_dispatch]
+
+permissions:
+  contents: write
+
+jobs:
+  update-release-branch:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Configure Git
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+      
+      - name: Prepare a Blank Release Branch
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git checkout --orphan release
+          git rm -rf .
+          git clean -fd  
+
+      - name: Prepare ZIP
+        # Fetch the latest release ZIP, extract, remove versiontage from folder structure, and re-zip it
+        run: |
+          RELEASE_TAG=$(curl -s https://api.github.com/repos/${{ github.repository }}/releases/latest | jq -r '.tag_name')
+          ZIP_URL=$(curl -s https://api.github.com/repos/DroneML/DroneML/releases/latest | jq -r '.zipball_url')
+          DOWNLOAD_ZIP="DroneML-${RELEASE_TAG}.zip"
+          curl -L -o ${DOWNLOAD_ZIP} -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" ${ZIP_URL}
+          unzip ${DOWNLOAD_ZIP} -d extracted_files
+          ls -l extracted_files
+          ORIGINAL_FOLDER=$(ls extracted_files)
+          mv extracted_files/${ORIGINAL_FOLDER} extracted_files/DroneML
+          ls -l extracted_files
+          mkdir -p DroneML
+          mv extracted_files/DroneML DroneML
```

Comment:
I got the following error while trying to install from zip:
![image](https://github.com/user-attachments/assets/b9a1c3e2-dbb1-431d-aab5-727847806541)
When I removed one level in the folder structure it did install correctly. So /DroneML instead of /DroneML/DroneML.
Please (double) check if you are in fact able to install the zip.
```suggestion
          mv extracted_files/DroneML .
```

Reply:
You are right, I added an extra layer here which breaks the installation. Your solution works.